components/dashboard/src/Menu.tsx

```diff
@@ -241,24 +235,29 @@ export default function Menu() {
     };
     const renderTeamMenu = () => {
         return (
-            <div className="flex p-1 pl-3 ">
+            <div className="flex p-1 pl-3">
                 {projectSlug && (
-                    <div className="flex h-full rounded-lg hover:bg-gray-100 dark:hover:bg-gray-800 px-2 py-1">
-                        <Link to={team ? `/t/${team.slug}/projects` : `/projects`}>
-                            <span className="text-base text-gray-600 dark:text-gray-400 font-semibold">
-                                {team?.name || userFullName}
-                            </span>
-                        </Link>
-                    </div>
+                    <Link to={team ? `/t/${team.slug}/projects` : `/projects`}>
+                        <span className="flex h-full text-base text-gray-500 dark:text-gray-400 dark:bg-gray-800 dark:hover:bg-gray-700 font-semibold pl-3 pr-2 py-1 bg-gray-50 hover:bg-gray-100 rounded-tl-2xl rounded-bl-2xl border-gray-100 dark:border-gray-700 border-r">
+                            {team?.name || userFullName}
+                        </span>
+                    </Link>
```

Comment:
**thought**: I swayed between having this also highlighted, following the pattern for the options on the right. Thoughts?

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-05-30 at 12 32 00 PM (2)" src="https://user-images.githubusercontent.com/120486/170963478-76c8bd17-2d69-4cd1-97d8-1b6376c450d0.png"> | <img width="1440" alt="Screenshot 2022-05-30 at 12 31 55 PM (2)" src="https://user-images.githubusercontent.com/120486/170963494-6e4f6f82-a947-42a0-ac3e-250a2024563a.png"> |

Cc @jankeromnes because we've also discussed navigation changes in the past in PRs like https://github.com/gitpod-io/gitpod/pull/5077, etc.

Reply:
To put this in perspective and side-by-side:

| BEFORE | PROPOSAL :a: (Current) | PROPOSAL :b: |
|-|-|-|
| <img width="1440" alt="170764390-4b6c26b2-c94b-42cf-b69a-d430f39a98df" src="https://user-images.githubusercontent.com/120486/171033901-9e62ad5d-f518-4a0d-a6b8-8b63873cf523.png"> | <img width="1440" alt="Screenshot 2022-05-30 at 12 32 00 PM (2)" src="https://user-images.githubusercontent.com/120486/170963478-76c8bd17-2d69-4cd1-97d8-1b6376c450d0.png"> | <img width="1440" alt="Screenshot 2022-05-30 at 12 31 55 PM (2)" src="https://user-images.githubusercontent.com/120486/170963494-6e4f6f82-a947-42a0-ac3e-250a2024563a.png"> |

Reply:
Resolving this and going with proposal :b: